media-sound/hydrogen/hydrogen-1.2.2.recipe

```diff
@@ -94,6 +93,8 @@ INSTALL()
 	mkdir -p $appsDir/Hydrogen
 	make -C build install
 	rm $libDir/libhydrogen-core-$portVersion.a
-	mv $binDir/hydrogen $appsDir/Hydrogen/Hydrogen
+	mv $binDir/hydrogen$secondaryArchSuffix $appsDir/Hydrogen/Hydrogen
```

Comment:
$secondaryArchSuffix isn't needed here ($binDir already sets the path to $prefix/bin/x86 on 32bit).
Build error on 32bit: https://build.haiku-os.org/buildmaster/master/x86_gcc2/logviewer.html?buildruns/5272/builds/82166.log